torch/nn/quantized/modules/linear.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from ...modules.module import Module
+from ...._jit_internal import weak_module
+
+
+
+@weak_module
+class Linear(Module):
+    r"""
+    A module that wraps the quantized fbgemm linear operator function
+    We adopt the same interface as `torch.nn.Linear`, please see https://pytorch.org/docs/stable/nn.html#torch.nn.Linear
+    for documentation.
+
+    Similar to `torch.nn.Linear`, attributes will be randomly initialized at
+        module creation time and will be overwritten later
+
+    Attributes:
+        _packed_weight: the weight that is first quantized and then packed using
+            fbgemm_linear_pack function
+        bias:   the quantized bias
+        output_scale: `scale` parameter of output Quantized Tensor
+        output_zero_point: `zero_point` parameter for output Quantized Tensor
+    """
+    def __init__(self, qweight, qbias, output_scale, output_zero_point):
+        super(Linear, self).__init__()
+        self.register_buffer('_packed_weight', torch.ops.quantized.fbgemm_linear_prepack(qweight))
+        self.register_buffer('output_scale', torch.Tensor([output_scale]))
+        self.register_buffer('output_zero_point', torch.Tensor([output_zero_point]))
+        self.register_buffer('bias', qbias)
+
+    def forward(self, x):
+        Y_q = torch.ops.quantized.fbgemm_linear(
+            x, self._packed_weight,
+            self.bias,
+            self.output_scale,
+            self.output_zero_point)
+        return Y_q
+
+    @staticmethod
+    def from_float(mod):
+        if hasattr(mod, 'qConfig'):
```

Comment:
also naming-wise, shall we call it q_config to follow python convention?

Reply:
maybe qconfig? since we have `QConfig` as a type.